packages/ui/src/SelectField.tsx

```diff
@@ -31,6 +31,12 @@ const Input = (props: any) => {
   return <components.Input {...props} autoComplete="chrome-off" />
 }
 
+// this component is implemented because we want to style things with the css (as opposed to the react-select recommended way of styling things via js).
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const MultiValue = (props: MultiValueProps<any>) => {
+  return <components.MultiValue {...props} className={cn({ [styles.multiValueIsFocused]: props.isFocused })} />
```

Comment:
@utkukaratas besides forcused state of the chip, should we style its default state? I would suggest the following:
- Use one of our neutral colors for its backgorund
- Use typography helpers (typographyBodySmall?) to use standard font sizes
- Use colorText for the text color
- Replace remove icon with the one from react-feather, so it matches what we use in our `Toast` component.
- Optionnally, try matching padding and margins with what we have in our `Toast`component

WDYT?

Reply:
we now use our IconButton for the button. resolving...

Reply:
@utkukaratas should we resolve all other proposals? I see that only the remove icon was changed.
I still suggest we do:
- Use one of our neutral colors for its backgorund
- Use typography helpers (typographyBodySmall?) to use standard font sizes
- Use colorText for the text color
- Try matching padding and margins with what we have in our Toastcomponent

WDYT?

Reply:
@aigoncharov i've used our IconButton instead of doing those manually. much cleaner IMO. it seems recent IconButton changes broke this PR but now that's fixed too.

Reply:
@utkukaratas I see that you used IconButton before. I am talking abou the color, padding and typography of the chip that represents a selected item. Currently, the chip does not match our styling.

Reply:
ah! i misunderstood that. will handle it...

Reply:
@aigoncharov 

- all of the issues above should be resolved. 
- it should be pixel perfect with the new design. (except some parts in the design those are not multiples of 4px)